cartography/intel/aws/iam.py

```diff
@@ -710,7 +710,7 @@ def sync_user_access_keys(
 
 @timeit
 def sync(
-    neo4j_session: neo4j.Session, boto3_session: boto3.session.Session, regions: str, account_id: str,
+    neo4j_session: neo4j.Session, boto3_session: boto3.session.Session, regions: List[str], account_id: str,
```

Comment:
Can we standardize on `current_aws_account_id`?

Reply:
Yeah, I was gonna do that in a future PR where I homogenize variable and function names